scripts/brackets.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+start Brackets.exe %*
```

Comment:
This doesn't work. If  we want to get this working we need to do something similar to what we did with MAC i.e. find the absolute path and the pass the absolute path to `start`. I think the script is taking the current `pwd`, from where the script was run, and then trying to find Brackets.exe in the parent folder of the that directory.

Since we are managing all of this through our Installer it should not be a problem doing a `start Brackets.exe %*` as the chances of someone editing the registry manually, are very less.